scripts/cubic/bootstrap-fmriprep.sh

```diff
@@ -298,7 +306,7 @@ singularity run --cleanenv -B ${PWD} \
     inputs/data \
     prep \
     participant \
-    -w ${PWD}/.git/tmp/wkdir \
+    -w ${PWD}/.git/tmp/wkdir
```

Comment:
I think removing this backslash makes fmriprep run without the additional flags in FMRIPREP_OPT_FILE?